include/user_functions/DropletVelocityAuxFunction.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef DropletVelocityAuxFunction_h
+#define DropletVelocityAuxFunction_h
+
+#include <AuxFunction.h>
+
+#include <vector>
+
+namespace sierra {
+namespace nalu {
+
+class DropletVelocityAuxFunction : public AuxFunction
+{
+public:
+  DropletVelocityAuxFunction(const unsigned beginPos, const unsigned endPos);
+
+  virtual ~DropletVelocityAuxFunction() {}
+
+  using AuxFunction::do_evaluate;
+  virtual void do_evaluate(
```

Comment:
Kind of unfortunate we call this with 2 verbs ;) Probably not relevant to this PR.

Reply:
It is strange for sure. There are a few places where patterns like this show up. There is also stuff like EquationSystem.C vs EquationSystems.C which can be confusing when working in the code. Likely worth a pass through the code to fix them down the road.